corehq/apps/export/urls.py

```diff
@@ -169,9 +169,6 @@
         DailySavedExportPaywall.as_view(),
         name=DailySavedExportPaywall.urlname),
 
-    url(r"^custom/dailysaved/download/(?P<export_instance_id>[\w\-]+)/$",
-        download_daily_saved_export,
-        name="download_daily_saved_export"),
```

Comment:
Looks like the bookmark/saved links was the reason someone ended up adding two links for the same end point. :|